frontend/src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```

Comment:
Whoever made this file, can you describe its purpose? I'm having trouble figuring out whats going on

Reply:
It is a file already included when I first set up mapbox and react front end